action.yml

```diff
@@ -1,12 +1,29 @@
-name: cffconvert
-author: https://github.com/jspaaks
-description: Validate CITATION.cff and .zenodo.json and check consistency
+---
+author: "https://github.com/jspaaks"
 branding:
-  icon: check
   color: purple
-runs: 
-  using: 'docker'
-  image: 'Dockerfile'
+  icon: check
+description: "Validate CITATION.cff and .zenodo.json and check consistency"
+inputs:
+  DIFF_IGNORE_WHITESPACE:
+    default: "1"
+    description: "Ignore all whitespace when diffing the expected .zenodo.json with the actual .zenodo.json contents."
+    required: false
+  FAILURE_EXPECTED:
+    default: "0"
+    description: "In testing it can be useful to flip the exit codes for failure and success. Set to 1 for flipped definition; 0 or empty gives normal behavior."
+    required: false
+  WORKDIR:
+    default: ""
```

Comment:
maybe the default workdir should be "./"

Reply:
changed (to ``'.'``)